### PR TITLE
Make 'Remove addon' actually remove addons. Solves #9

### PR DESCRIPTION
**Pull request by shinnova**

### CODE REVIEW

File: modules/application.py [hunk 1]
```diff
@@ -349,6 +350,20 @@ def removeAddon(self):
...
+            addonName =  str(self.addonList.item(row, 0).text())
+            for item in contents:
+                itemDir = "{}/{}".format(parent, item)
+                if os.path.isdir(itemDir) and not item.lower().startswith("blizzard_"):
+                    toc = "{}/{}.toc".format(itemDir, item)
+                    if os.path.exists(toc):
+                        tmp = self.extractAddonMetadataFromTOC(toc)
+                        if re.search(tmp[0], addonName) != None:
+                            rmtree(itemDir)
+            if addonName == "InFlight Taxi Timer":
```
Comment (oleerik) [inline modules/application.py]: Having special exceptions doesn't seem to be a good idea. Maybe have an error pop-up about "couldn't delete addon" instead for all failed deletions?

**Comment by oleerik** [comment]
Note: This one is **not ready** for merging just yet

**Comment by shinnova** [comment]
Indeed not yet ready for merging. Still needs some work to be done.

**Comment by ephraim** [comment]
One way to me sure you can't delete anything which isn't from that addon would be, to download the addon and walk exactly the content of the zip and delete it in the Addons folder.
But I don't like this either.
Or create a txt file with all file-paths in it, when installing it via lcurse. So you can remove these addons you have updated one via lcurse.

**Pull request closed without merging**